posthog/temporal/tests/batch_exports/utils.py

```diff
@@ -35,3 +38,29 @@ async def get_record_batch_from_queue(queue, produce_task):
 
         return record_batch
     return None
+
+
+def get_flaky_clickhouse_client(*args, fail_after_records: int, **kwargs):
+    return FlakyClickHouseClient(*args, fail_after_records=fail_after_records, **kwargs)
```

Comment:
Seems kind of pointless, what's wrong with just using the class?

Reply:
oh yes, I was playing around with this and ended up refactoring it, so indeed this is no longer needed. Have cleaned it up now, thanks